arch/Kconfig

```diff
@@ -531,6 +531,13 @@ config FP_SHARING
 	  This option allows multiple threads to use the floating point
 	  registers.
 
+config FLOAT_64BIT
+	bool "Double precision floating point"
+	depends on FLOAT
+	depends on RISCV
+	help
+	  This option means that floating point registers are 64bit width.
```

Comment:
In ARM, this is controlled by the `CPU_HAS_FPU_DOUBLE_PRECISION` symbol:
https://github.com/zephyrproject-https://github.com/zephyrproject-rtos/zephyr/blob/6492dcec6fa159aa9fb794d69c6abf8fc2b68203/soc/arm/Kconfig#L60

I wonder if we should move that symbol out of `soc/arm` and use that here instead.

Reply:
by the way, maybe better to write "64bit-wide"

otherwise, yes, that is what i think as well - we should try to unify the Floating-point configuration across architectures

@xodus7 @stephanosio dou you think we could define CPU_HAS_FPU_DOUBLE_PRECISION right under CPU_HAS_FPU definition?

Reply:
> dou you think we could define CPU_HAS_FPU_DOUBLE_PRECISION right under CPU_HAS_FPU definition?

It sounds like that would be the best place to add it.